### PR TITLE
docs(site): Add metaphor-free TL;DR  to Introduction

### PR DESCRIPTION
**Pull request by joriswitteman**
I had a lot of trouble understanding this page because it immediately forces me into a mental model of 'mutants' and 'mutations' that only serve to distract me from understanding the gist of mutation testing. Mutation itself is even confusing because it make me think of network calls that do data mutations. It also confuses me because it reminds me of mutable vs immutable approaches in code.

This TL;DR summarises the page in a neutral way, without references to metaphors, that I believe would have made me got the concept right away.

### CODE REVIEW

**Comment by rachied** [comment]
> Mutation testing changes the return statements in your functions

 I think this simplifies things a bit too much. Mutations can and do occur outside of return statements.

**Comment by joriswitteman** [comment]
> > Mutation testing changes the return statements in your functions
> 
> I think this simplifies things a bit too much. Mutations can and do occur outside of return statements.

How about:

> **TL; DR: Mutation testing changes little parts of your code to change the outcome, to verify that the tests that cover them, actually fail properly as a result.**

**Comment by rouke-broersma** [comment]
@nicojs I'm not sure what to think of this. Personally I think the text we have now is fine, however we have a clear user request for clearer language so.. It's probably bias and we should do something with this. 


> Mutation testing introduces changes to your code, then runs your unit tests against the changed code. It is expected that your unit tests now fail. If they don't fail for a change in the code this indicates the code is not sufficiently covered by your tests.

**Comment by joriswitteman** [comment]
@rouke-broersma What you're citing there sounds perfect to me. That isn't yet on the site, is it?

**Comment by rouke-broersma** [comment]
No it's my proposal.

File: docs/General/introduction.mdx [hunk 1]
```diff
@@ -5,6 +5,8 @@ slug: '/'
 sidebar_position: 1
 ---
 
+**TL; DR: Mutation testing introduces changes to your code, then runs your unit tests against the changed code. It is expected that your unit tests now fail. If they don't fail for a change in the code this indicates the code is not sufficiently covered by your tests.**
```
Comment (nicojs) [inline docs/General/introduction.mdx]:
```suggestion
**TL; DR: Mutation testing introduces changes to your code, then runs your unit tests against the changed code. It is expected that your unit tests will now fail. If they don't fail, it might indicate your tests do not sufficiently cover the code.**
```

This is a bit more conservative. What do you think?

**Review by nicojs — CHANGES_REQUESTED** [review]
Thanks for your feedback. What do you think about my suggestion?

File: docs/General/introduction.mdx (hunk 1 above)
Comment (rouke-broersma) [reply docs/General/introduction.mdx]: Yep I agree with going more conservative.
